extensions/claude-code-launcher/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "claude-code-launcher",
+  "title": "Claude Code Launcher",
+  "description": "Quickly open Claude Code in any directory with your preferred terminal.",
+  "icon": "icon.png",
+  "author": "stephendolan",
+  "categories": ["Developer Tools", "Productivity"],
+  "license": "MIT",
+  "version": "1.0.0",
+  "commands": [
+    {
+      "name": "open-claude-code",
+      "title": "Open Project",
+      "subtitle": "Claude Code Launcher",
+      "description": "Launch Claude Code in favorite directories.",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "claudeBinaryPath",
+      "title": "Claude Binary Path",
+      "description": "Path to the Claude executable",
+      "type": "textfield",
```

Comment:
Could be better to make this a directory?

```suggestion
      "type": "directory",
```

Reply:
Ah, that's great. Thanks for the suggestion, @pernielsentikaer